fr/_posts/2021-07-14-liens-plus-efficaces.md

```diff
@@ -0,0 +1,105 @@
+---
+altLangPage: "/2021/07/14/effective-links"
+date: 2021-07-14
+description: " "
+title: "3 étapes pour rendre les liens plus efficaces"
+---
+
+Sur le Web, les internautes sont à la recherche de réponses. La manière dont nous configurons une page influence la façon dont les internautes se déplacent dans notre contenu. Lorsque nous ajoutons des liens, nous pouvons fournir davantage de contexte ou amener les internautes à l'étape suivante d'un processus sans alourdir une page avec trop de renseignements. 
+
+Un lien bien placé peut fournir une «&nbsp;piste d'information&nbsp;» que les gens peuvent suivre facilement pour accomplir une tâche. Mais l'ajout d'un mauvais lien au mauvais endroit peut envoyer les gens dans des boucles frustrantes et les empêcher d'utiliser votre page Web avec succès.
+
+Lorsque vous concevez un contenu comprenant des liens, concentrez-vous sur les trois clés du succès :
+
+1. Quand ajouter des liens
+2. Où placer les liens
+3. Comment rédiger un bon texte de lien
+
+## Tous les liens sont des appâts à clics
+
+Ce n'est pas parce que nous pouvons ajouter des liens partout dans notre contenu que nous devrions le faire. 
+
+Les liens agissent comme des panneaux de signalisation pour les bretelles d'accès sur une autoroute. Ils guident les utilisateurs. Ils attirent également l'attention. Les gens supposent qu'un lien est là pour être suivi.
+
+Imaginez une autoroute à 8 voies très fréquentée avec des panneaux pointant dans toutes les directions. C'est déroutant, non ? Dois-je rester sur ma voie ou prendre la bretelle de sortie ?! 

```

Comment:
What’s our stance on spaces before question/exclamation marks?

Reply:
No space before question and exclamation marks.

Reply:
```suggestion
Imaginez une autoroute à 8 voies très fréquentée avec des panneaux pointant dans toutes les directions. C'est déroutant, non? Dois-je rester sur ma voie ou prendre la bretelle de sortie?! 

```